scripts/bs-wrapper.js

```diff
@@ -52,13 +46,15 @@ function hasRunningBuild() {
 }
 
 function runTests() {
-  return new Promise((resolve) => {
-    const [command, ...args] = process.argv.slice(2)
-    const commandProcess = spawn(command, args, { stdio: 'inherit' })
-    commandProcess.on('exit', resolve)
-  })
+  const [command, ...args] = process.argv.slice(2)
+  return spawnCommand(command, args)
 }
 
 function timeout(ms) {
   return new Promise((resolve) => setTimeout(resolve, ms))
 }
+
+main().catch((error) => {
+  printError('\nStacktrace:\n', error)
+  process.exit(1)
+})
```

Comment:
We are doing that a lot, what about extracting it also?
maybe something like 
```ts
printFormattedError(error)
process.exit(1)
```
or
```ts
catch(logAndExit)
```

Reply:
Yes, sure. I'll do the second proposition